src/api/RetryType.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
non disabilitare regole di eslint dove non è necessario.
per ```Article.ts``` è sensato dato che la forma dell'oggetto è data dal backend (che magari provo a convincere a usare camelCase lol), ma qua ha più senso aggiornare la configurazione di eslint per l'utilizzo degli enum.

direi di aggiungere la regola
```js
{
    selector: "enumMember",
    format: ["UPPER_CASE"],
},
```
in fondo a ```@typescript-esling/naming-conventions``` in ```.eslintrc.js```

Reply:
Okay nice